src/pages/workspaces/workspace/Data.js

```diff
@@ -280,53 +288,55 @@ const BucketContent = _.flow(
       )
     ]),
     div({ style: { margin: '1rem -1rem 1rem -1rem', borderBottom: `1px solid ${colors.dark(0.25)}` } }),
-    h(SimpleTable, {
-      'aria-label': 'file browser',
-      columns: [
-        { header: div({ className: 'sr-only' }, ['Actions']), size: { basis: 24, grow: 0 }, key: 'button' },
-        { header: h(HeaderCell, ['Name']), size: { grow: 1 }, key: 'name' },
-        { header: h(HeaderCell, ['Size']), size: { basis: 200, grow: 0 }, key: 'size' },
-        { header: h(HeaderCell, ['Last modified']), size: { basis: 200, grow: 0 }, key: 'updated' }
-      ],
-      rows: [
-        ..._.map(p => {
-          return {
-            button: div({ style: { display: 'flex' } }, [
-              icon('folder', { size: 16, 'aria-label': 'folder' })
-            ]),
-            name: h(TextCell, [
-              makeBucketLink({
-                label: p.slice(prefix.length),
-                target: `gs://${bucketName}/${p}`,
-                onClick: () => load(p),
-                'aria-label': `${p.slice(prefix.length, -1)} (folder)`
-              })
-            ])
-          }
-        }, prefixes),
-        ..._.map(({ name, size, updated }) => {
-          return {
-            button: h(Link, {
-              style: { display: 'flex' }, onClick: () => setDeletingName(name),
-              tooltip: 'Delete file'
-            }, [
-              icon('trash', { size: 16, className: 'hover-only' })
-            ]),
-            name: h(TextCell, [
-              makeBucketLink({
-                label: name.slice(prefix.length),
-                target: `gs://${bucketName}/${name}`,
-                onClick: () => setViewingName(name),
-                'aria-haspopup': 'dialog',
-                'aria-label': `${name.slice(prefix.length)} (file)`
-              })
-            ]),
-            size: filesize(size, { round: 0 }),
-            updated: Utils.makePrettyDate(updated)
-          }
-        }, objects)
-      ]
-    }),
+    div({ style: { flex: '1 1 0', overflow: 'auto' }}, [
```

Comment:
Interestingly, `}}` throws a lint warning and yet somehow did not fail the lint step in ci.

Reply:
Yeah, since all these stylistic things are configured as warnings instead of errors, CI will allow them in.
https://github.com/DataBiosphere/terra-ui/blob/6af15805aeed7adc03be7101867d5eb501654b42/.eslintrc.js#L36

Reply:
Oh, I thought or some reason that the "LGTM" stage caught indenting problems. Anyway, I'm fixing the indenting in my PR.

Reply:
Opened #2870 to prevent stuff like this from making it into `dev`.